.gitattributes

```diff
@@ -2,4 +2,3 @@
 
 .gitattributes export-ignore
 .gitignore export-ignore
```

Comment:
I don't know how `.gitattributes` works but I am sure you are doing it correctly.

Reply:
I am. That will make it so a distribution download will not include either of those files. I think we should include the composer.json

Reply:
Yes include it.

Reply:
.gitattributes works to exclude files/folders that you don't want downloaded in a distribution of the project.
.gitignore works to exclude files/folders from being in the project entirely.